app/views/rating_questions/edit.html.erb

```diff
@@ -0,0 +1,20 @@
+
+<div class="form"><%= form_with model: @rating_question, url: rating_question_path,local: true do |f| %>
```

Comment:
Move the `form_with` tag to its own line and indent it within the `<div class="form">`

Reply:
no probs